### DISCUSSION TITLE
How can I install the package of verdaccio local services using bit install?

### DISCUSSION THREAD

**Discussion by AmelloAster** [teambit/bit#7406] [FAQ]
How can I install the package of verdaccio local services using bit install?

**Comment by itaymendel** [comment]
if you are using bit.cloud to host your components, you can use verdaccio to proxy all that starts with `@orgname`, similar to how you proxy for npmjs.org.
if you self-host, you need to have bit publish components to your verdaccio instance - https://bit.dev/reference/packages/publishing-components-to-commonjs-registries
